BUILDING.md

```diff
@@ -108,11 +108,16 @@ git submodule update --init
 
 #### Building the projects
 
-Open `rpcs3.sln`. The recommended build configuration is `Release - LLVM` for all purposes.
+Open `rpcs3.sln`. The recommended build configuration is `Release` for all purposes.
```

Comment:
```suggestion
Open `rpcs3.sln`. The recommended build configuration is `Release` for all purposes. (On older revisions: `Release - LLVM`)
```

Reply:
I think the line can be rewritten, actually. I added the recommendation during a time when the debug builds were broken, so thats what I meant by "all purposes". Maybe just replace it with the note about older releases.

Reply:
I'd just drop the "for all purposes"